contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,99 @@
+Reporting Guidelines
+====================
+
+If you believe someone is violating the Code of Conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** The privacy of everyone included in
+the report is of our highest concern. Second to privacy there is transparency.
+After every report we will determine if a public statement should be made. If
+that's the case, the identities of all victims, reporters, and the accused will
+remain confidential unless those individuals instruct us otherwise. The details
+of the incident may also be generalized.
+
+If you believe anyone is in physical danger, please notify appropriate emergency
+services first by calling the relevant local authorities. If you are unsure what
+service or agency is appropriate to contact, include this in your report and we
+will attempt to notify them.
+
+In your report please include:
+
+  * Your contact info for follow-up contact.
+  * Names (legal, nicknames, or pseudonyms) of any individuals involved.
+  * If there were other witnesses besides you, please try to include them as well.
+  * When and where the incident occurred. Please be as specific as possible.
+  * Your account of what occurred.
+  * If there is a publicly available record (e.g. a mailing list archive or a
+    public IRC or Slack log), please include a link.
```

Comment:
I would recommend a link + a screenhost or a local archive (HTML page), just in case the contents are tampered with afterwards, edited or even removed.